shared/authorize.py

```diff
@@ -1,38 +0,0 @@
-from functools import wraps
```

Comment:
We need this file for token authentication. I am not sure why it's removed in the PR